packages/flutter/lib/utils/key_manager.dart

```diff
@@ -196,10 +199,11 @@ class KeyManager {
 
   /// [signMessage]
   /// External function that does the actual signing
-  Future<MsgSignature> signMessage(Uint8List message,
+  Future<String> signMessage(Uint8List message,
       {int? index, String? alias}) async {
     final privKey = await _getPrivateKey(index ?? 0, alias: alias);
-    final signature = privKey.signToEcSignature(message);
+    final encodePrivKey = base64Url.encode(privKey.privateKey);
```

Comment:
Don't encode the private key, there's a method in there to get the string representation

Reply:
hexlify?